Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:17-jdk-alpine
```

Comment:
suggestion: maybe we can create a folder called: docker then put every docker related files there. e.g. docker-compose files, etc.

Reply:
I think the dockerfile should reside on the root level (same level as build result -> target folder), what do you think?

Reply:
makes sense.

Reply:
Can we merge this PR? @skayikci @halilturkoglucs

Reply:
I can merge as soon as @skayikci approves the pr